api/integrations/hg/zulip-changegroup.py

```diff
@@ -101,13 +105,15 @@ def send_zulip(email, api_key, site, stream, subject, content):
     client.send_message(message_data)
 
 def get_config(ui, item):
+    # type: (Any, str) -> Optional[str]
```

Comment:
You can check the type of `ui` in [Mercurial API](https://www.mercurial-scm.org/wiki/MercurialApi).